VERSION-2-MIGRATION.md

```diff
@@ -0,0 +1,15 @@
+# Vibe 2 Migration
+
+## What's changed?
```

Comment:
supported browsers
importing only from the root index file

Reply:
browsers list didn't get updated, though we wanted it originally, right?

Reply:
What about importing only from the root index file?

Reply:
browsers list - then let's do it. It should be only in the docs so no big change here.
index file - to prevent users from using `/dist/src` and simply entries

Reply:
and also add that we no longer export the global class names, meaning only components should be used